src/GenFacades.Desktop/project.json

```diff
@@ -0,0 +1,40 @@
+﻿{
+    "dependencies": {
+        "System.Collections": "4.0.10-beta-*",
```

Comment:
I assume these are all dependencies that Microsoft.CCI.Extensions uses? At first glance it definitely looks like more than is necessary. I'm pretty sure that CCI.Extensions doesn't use Immutable or Concurrent or AppContext for example. I also thought you removed the console dependency.